init/setup.js

```diff
@@ -70,11 +70,13 @@
 	}
 	console.log("Database credentials setup successfully.");
 
-	let packageManager = null;
```

Comment:
the entire file should be reverted, as we use `auto-setup` for working with env values

Reply:
done